### PR TITLE
Added EXO P2 requirement to plan-cloud-voicemail.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7438.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbHybrid/hybrid/plan-cloud-voicemail.md [hunk 1]
```diff
@@ -79,6 +79,8 @@ In addition to the requirements above, the below requirements must be configured
 
 - If you have an on-premises Exchange server, set up Cloud Voicemail using the steps in [Set up Cloud Voicemail for Exchange Server Mailbox Users](/microsoftteams/set-up-phone-system-voicemail#set-up-cloud-voicemail-for-exchange-server-mailbox-users).
 
+- If the user has a mailbox on Exchange Online, an Exchange Online Plan 2 is required.
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/plan-cloud-voicemail.md:82]:
```suggestion
- If the user has a mailbox in Exchange Online, an Exchange Online Plan 2 is required. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request closed without merging**